django/db/models/base.py

```diff
@@ -1333,12 +1333,17 @@ def _get_next_or_previous_in_order(self, is_next):
             setattr(self, cachename, obj)
         return getattr(self, cachename)
 
-    def _get_field_value_map(self, meta, exclude=None):
+    def _get_field_expression_map(self, meta, exclude=None):
```

Comment:
@charettes we probably no longer need to do this since DatabaseDefault is handling the logic… or do we prefer to leave it in?

Reply:
I haven't run the branch locally but from my understanding the logic below was still required as we wanted

1. To have literal value wrapped in `Value`
2. To have non-literal expressions, such as `DatabaseDefault`, be returned as is

Reply:
Ah yes I see now `Value` does its own `as_sql()` so we do still need it 👍

Reply:
As a side note I feel like I've seen this optional wrapping pattern a few times I wonder if there's any benefit to making a factory method along the lines of `Value.wrap_if_not_expression(value_maybe_expression)`

Reply:
Possibly but since this will be backported it should be done in another PR.